eppo_core/src/background/runtime.rs

```diff
@@ -101,6 +101,7 @@ impl BackgroundRuntime {
 
     /// Command background activities to stop and exit.
     pub(crate) fn stop(&self) {
+        log::debug!(target: "eppo", "stopping background runtime");
```

Comment:
Did you intend to keep this debug here?

Reply:
Absolutely: https://github.com/Eppo-exp/eppo-multiplatform/pull/197/commits/5353290c3de41dfbd6a6d61f66800673c3cb6e4b

I've spent a couple of days debugging why poller mysteriously crashes. That log line would cut the debugging to one minute 🙈